lib/tasks.js

```diff
@@ -271,6 +271,19 @@ module.exports = async options => {
       await run(`!rm -rf ${dist.stageDir}`);
     }
 
+    if (github.release) {
+      getSpinner().stopAndPersist({
+        symbol: `❗`,
+        text: `To Github repository ${config.options.repo.remote}`
```

Comment:
* I'd prefer a simple `🔗 [url]` here. I think it's clear from the url alone what we're linking to.
* The url should be coming from the return value of `release`: `releaseInfo.html_url` (not the repo url, which could be a non-web url, but the newly created release page).